### ISSUE TITLE
install-node.sh update URL

### ISSUE THREAD

**Issue by wcjochem** [rstudio/shiny-server#425]
Hi all,

I installed from source recently and had trouble with shiny-server/external/node/install-node.sh. On closer inspection, the script points to this base path: "https://github.com/jcheng5/node-centos6/releases/download/" which returns 404 not found.

For my install, I was able to work around by repointing to https://nodejs.org/en/download/ for the appropriate version and SHA key found at the same page. Is there a better way?

**Comment by jcheng5** [comment]
What happens when you try to run `install-node.sh`? It works fine for me. It points to the releases available here: https://github.com/jcheng5/node-centos6/releases/ (I did notice however that the "Building Shiny Server from source" instructions didn't include instructions to run this script, so I just added it)

There's no problem with using the official Node.js binaries, they just won't work on CentOS/RedHat 6 which is still a requirement for some of our customers.
